.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: main
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: DerYeger/yarn-setup-action@v1.0.1
+        with:
+          node-version: 15.9.0
+  build:
+    runs-on: ubuntu-latest
+    needs: setup
+    steps:
+      - uses: DerYeger/yarn-setup-action@v1.0.1
+        with:
+          node-version: 15.9.0
```

Comment:
I smell a feature request, that `yarn-setup-action` respect `.nvmrc`.

Reply:
Also, should we be testing this library across matrix of versions?

Reply:
Tada https://github.com/DerYeger/yarn-setup-action/issues/9

Reply:
I don't have strong opinions on the `matrix` argument. I'd say no for now, since this library has no known users outside of Freckle.

Reply:
> I'd say no for now, since this library has no known users outside of Freckle.

I agree. I might consider letting the action do whatever it does (latest?) instead of pinning to an arbitrary version we've decided to use in our own app, but no big deal either way.